File: programme/add-two-numbers/AddTwoNumbers.java
// Write a program to add two numbers

import java.util.Scanner;

public class AddTwoNumbers {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a, b;

        System.out.print("Enter two numbers to add : ");
        a = sc.nextInt();
        b = sc.nextInt();

        int sum = a + b;
        System.out.println("\nThe sum of numbers is : " + sum);

        sc.close();
    }
}

File: programme/add-two-numbers/AddTwoNumbers.kt
// Write a programme to add two numbers

import java.util.Scanner

class AddTwoNumbers {

    fun main() {
        val sc = Scanner(System.`in`)

        print("Enter two numbers to add : ")
        val a = sc.nextInt()
        val b = sc.nextInt()

        val sum = a + b
        println("\nThe sum of numbers is : $sum")
        sc.close()
    }
}